### PR TITLE
Example app integration tests

### PR DESCRIPTION
**Pull request by jaceko**

### CODE REVIEW

File: example/example-command/example-command-handler/src/main/java/uk/gov/justice/services/example/cakeshop/command/handler/AddRecipeCommandHandler.java [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
     private static final Logger LOGGER = getLogger(MakeCakeCommandHandler.class);
     private static final String FIELD_RECIPE_ID = "recipeId";
+    public static final String EVENT_NAME = "cakeshop.events.recipe-added";
```
Comment (mapingo) [inline example/example-command/example-command-handler/src/main/java/uk/gov/justice/services/example/cakeshop/command/handler/AddRecipeCommandHandler.java]: Can EVENT_NAME be private

File: example/example-it/src/test/java/uk/gov/justice/services/example/cakeshop/it/CakeShopIT.java [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    private static final int ACCEPTED = 202;
+    private static final String RECIPES_RESOURCE_URI = "http://localhost:8080/example-command-api/command/api/rest/cakeshop/recipes/";
+    private static final String CAKES_RESOURCE_URI = "http://localhost:8080/example-command-api/command/api/rest/cakeshop/cakes/";
+    private static final String ADD_RECIPE_MEDIA_TYPE = "application/vnd.cakeshop.commands.add-recipe+json";
+    private static final String MAKE_CAKE_MEDIA_TYPE = "application/vnd.cakeshop.commands.make-cake+json";
+    private static final String LIQUIBASE_EVENT_STORE_DB_CHANGELOG_XML = "liquibase/event-store-db-changelog.xml";
+    public static final String H2_DRIVER = "org.h2.Driver";
+    public static final String DB_URL = "db.url";
+    public static final String DB_USER_NAME = "db.userName";
+    public static final String DB_PASSWORD = "db.password";
```
Comment (mapingo) [inline example/example-it/src/test/java/uk/gov/justice/services/example/cakeshop/it/CakeShopIT.java]: The above public constants should be made private

File: example/example-command/example-command-handler/src/main/java/uk/gov/justice/services/example/cakeshop/command/handler/AddRecipeCommandHandler.java (hunk 1 above)
Comment (jaceko) [inline example/example-command/example-command-handler/src/main/java/uk/gov/justice/services/example/cakeshop/command/handler/AddRecipeCommandHandler.java]: fixed

File: example/example-it/src/test/java/uk/gov/justice/services/example/cakeshop/it/CakeShopIT.java (hunk 2 above)
Comment (jaceko) [inline example/example-it/src/test/java/uk/gov/justice/services/example/cakeshop/it/CakeShopIT.java]: fixed

**Pull request merged**